server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java

```diff
@@ -57,4 +57,46 @@ public void testRPUSHX() {
             .hasMessageContaining("ERRWRONGTYPE");
    }
 
+   public void testLPUSH() {
```

Comment:
Should we verify the order, too?

Reply:
not here. we need it in another PR with other operations